TypeCobol/Compiler/Sql/Model/SqlColumnName.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.IO;
+using TypeCobol.Compiler.CodeElements;
+
+namespace TypeCobol.Compiler.Sql.Model
+{
+    public class SqlColumnName : SqlExpression
+    {
+        public SqlColumnName(SymbolReference symbol)
+        {
+            this.Symbol = symbol;
+        }
+
+        public SymbolReference Symbol { get; }
+
+        protected override void DumpContent(TextWriter output, int indentLevel)
+        {
+            DumpProperty(output, nameof(Symbol), Symbol, indentLevel);
+        }
+
+        protected override bool VisitSqlObject(ISqlVisitor visitor)
+        {
+            return visitor.Visit(this) ;
+        }
+    }
```

Comment:
Could you please move this into SqlExpression.cs file (and delete this file) ? We already have constants defined there, so I think we'll put all expressions in the same file.